CHANGELOG.md

```diff
@@ -4,22 +4,33 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## To be released
+## 1.7.1 (2020-05-15)
 
-**Major Features**
+This maintenance release contains bugfixes since the 1.7.0 release. We deem it medium
+priority for upgrading.
+
+In particular the fixes contained in this maintenance release address bugs in continuous
+aggregates with real-time aggregation and PostgreSQL 12 support.
 
 **Bugfixes**
+* #1834 Define strerror() for Windows
```

Comment:
Should probably add https://github.com/timescale/timescaledb/pull/1884 and https://github.com/timescale/timescaledb/pull/1869 to this list.

Reply:
#1869 isn't merged yet and has changelog entry in the PR itself, added #1884